plugins/modules/api_info.py

```diff
@@ -0,0 +1,268 @@
+      assignment_group: 8a4dde73c6112278017a6a4baf547aa7
+      business_duration: "1970-01-20 05:38:50"
+      business_service: ""
+      business_stc: "1661930"
+      calendar_duration: "1970-03-21 20:38:50"
+      calendar_stc: "6899930"
+      caller_id: 681ccaf9c0a8016400b98a06818d57c7
+      category: inquiry
+      caused_by: ""
+      child_incidents: ""
+      close_code: Solved (Work Around)
+      close_notes: Gave workaround
+      closed_at: "2020-07-07 23:18:40"
+      closed_by: 9ee1b13dc6112271007f9d0efdb69cd0
+      cmdb_ci: ""
+      comments: ""
+      comments_and_work_notes: ""
+      company: 31bea3d53790200044e0bfc8bcbe5dec
+      contact_type: phone
+      contract: ""
+      correlation_display: ""
+      correlation_id: ""
+      delivery_plan: ""
+      delivery_task: ""
+      description: Noticing today that any time I send an email with an attachment, it
+        takes at least 20 seconds to send.
+      due_date: ""
+      escalation: "0"
+      expected_start: ""
+      follow_up: ""
+      group_list: ""
+      hold_reason: ""
+      impact: "1"
+      incident_state: "7"
+      knowledge: "false"
+      location: ""
+      made_sla: "false"
+      notify: "1"
+      number: INC0000013
+      opened_at: "2020-07-06 23:15:58"
+      opened_by: 9ee1b13dc6112271007f9d0efdb69cd0
+      order: ""
+      parent: ""
+      parent_incident: ""
+      priority: "1"
+      problem_id: ""
+      reassignment_count: "2"
+      reopen_count: ""
+      reopened_by: ""
+      reopened_time: ""
+      resolved_at: "2020-09-24 19:54:48"
+      resolved_by: 6816f79cc0a8016401c5a33be04be441
+      rfc: ""
+      route_reason: ""
+      service_offering: ""
+      severity: "3"
+      short_description: EMAIL is slow when an attachment is involved
+      sla_due: ""
+      state: "7"
+      subcategory: ""
+      sys_class_name: incident
+      sys_created_by: don.goodliffe
+      sys_created_on: "2020-07-07 23:18:07"
+      sys_domain: global
+      sys_domain_path: /
+      sys_id: 46cebb88a9fe198101aee93734f9768b
+      sys_mod_count: "5"
+      sys_tags: ""
+      sys_updated_by: VALUE_SPECIFIED_IN_NO_LOG_PARAMETER
+      sys_updated_on: "2020-09-24 19:54:48"
+      task_effective_number: INC0000013
+      time_worked: ""
+      universal_request: ""
+      upon_approval: ""
+      upon_reject: ""
+      urgency: "1"
+      user_input: ""
+      watch_list: ""
+      work_end: ""
+      work_notes: ""
+      work_notes_list: ""
+      work_start: ""
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+
+from ..module_utils import arguments, client, errors, table, utils
+from ..module_utils.api import (
+    transform_query_to_servicenow_query,
+    POSSIBLE_FILTER_PARAMETERS,
+    table_name,
+    FIELD_COLUMNS_NAME,
+    FIELD_SYS_ID,
+    get_query_by_sys_id,
+)
+
+
+def run(module, table_client):
+    if FIELD_COLUMNS_NAME in module.params:
+        module.params[FIELD_COLUMNS_NAME] = ",".join(
```

Comment:
I would advise against assigning anything to the `module.params` dictionary. One reason is that when ansible-playbook is run in verbose mode, the invocation parameters are added to the output, i.e., the exact set of parameters as described in the playbook. A user expects to see there the parameters as specified in the playbook, but since the module modifies the value, this could lead to confusion.

Reply:
Ok, thanks for explaining. Resolved by:
- https://github.com/ansible-collections/servicenow.itsm/pull/179/commits/8b8f8c687447f9d2d7ab15c90b4eceb0e9b8d314 (which included operator for merging dicts `|`, which isn's supported for all python versions)
- https://github.com/ansible-collections/servicenow.itsm/pull/179/commits/9e2c4a7961c8a09e9dfe2136fc10b8847ec280e8 (Fixed the non-supported `|`).